Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -262,6 +262,8 @@ public string FormatDisplayValue( TItem item )
         /// </summary>
         [Parameter] public RenderFragment<CellEditContext> EditTemplate { get; set; }
 
+        [Parameter] public object DefaultValue { get; set; }
```

Comment:
I had in mind for this to be function so it can be set dynamically, like `Func<TItem, object>`.

Reply:
True. I will change it. :)

Reply:
When we use `Func<TItem, object>`, user could set properties on them self.
Then we could move it to DataGrid and use `Func<TItem>` with name of SetDefaultValues.
Like this:
```
<DataGrid ... SetDefaultValues="employee => {
  employee.Salary = 100.0;
  employee.IsActive = true;
}"
```

Reply:
It might be good idea to have `Func<TItem>` on DataGrid but maybe implementing it could be tricky. Try and see if i'ts not too complicated.

Otherwise just have `Func<object>` on DataGridColumn. It's way easier I think.

Reply:
How do you think about actual version?

Reply:
Seems good now. Still not sure about the name `SetDefaultValues`. Maybe `NewItemDefaultValue` would better indicate the meaning of it? Or do you have better name in mind?

Other than that, the code is nice and clean.

Reply:
Hmm, difficult. `SetDefaultValues` call, what the action should do it. `NewItemDefaultValue` call, when action is executed. Maybe `NewItemDefaultSetter`?

Reply:
I like `NewItemDefaultSetter`. We can go with that name.